### DISCUSSION TITLE
How to link to any location in another norg file?

### DISCUSSION THREAD

**Discussion by Gandharikos** [nvim-neorg/neorg#751] [Q&A]
just like org-roam and obsidian, how to do that?

**Comment by Gandharikos** [comment]
ok, I find in doc 
```
{:path:# content}
```

**Comment by noisyBrain** [comment]
Hi! Could you tell me where you found that? I'm reading in the wiki, but I couldn't find it.

Also, I have to ask: 
Now I trying to link a note that is on (e.g ~/neorg/personal/notes) with other that is on ~/neorg/work/tasks. Should I do `{:../../work/tasks/index:#content}`? 
Because, that is the only way I could do it and in vhyrro's video looks like much clean

**Reply by andreadev-it** [reply]
That also depends on your dirman module configuration, which handles the workspaces. If it's in the same workspace, you can link to the file using the path relative from the workspace root

**Reply by noisyBrain** [reply]
mmm I think I understand. Thanks!

**Comment by Gandharikos** [comment]
I found in neorg/doc/neorg.norg
```
    {* Heading 1}
    {** Heading 2}
    {*** Heading 3}
    {**** Heading 4}
    {***** Heading 5}
    {****** Heading 6}
    {******* Heading level above 6}
    {# Generic}
    {$ Definition}
    {^ Footnote}
    {:norg_file:}
    {:norg_file:* Heading 1}
    {:norg_file:** Heading 2}
    {:norg_file:*** Heading 3}
    {:norg_file:**** Heading 4}
    {:norg_file:***** Heading 5}
    {:norg_file:****** Heading 6}
    {:norg_file:******* Heading level above 6}
    {:norg_file:# Generic}
    {:norg_file:* Marker}
    {:norg_file:$ Definition}
    {:norg_file:^ Footnote}
    {https://github.com/}
    {file:///dev/null}
    {/ external_file.txt}
    Note that the following links are malformed:
    {:norg_file:/ external_file.txt}
    {:norg_file:https://github.com/}
```

**Reply by noisyBrain** [reply]
Very useful, thanks!

**Comment by pysan3** [comment]
One alias I find useful is you can use `$/` to point to the root of the workspace. 

So, if you write `{:$/zk/foo:}` in workspace `~/norg/work`, this points to `~/norg/work/zk/foo.norg` from **any file** inside the same workspace (like an absolute path). 

But I don't think there's an easy way to link to files outside of the same workspace.

**Reply by vhyrro** [reply]
Hey, just FYI you can also add a workspace name after the `$` to link across workspaces: `{:$workspace_name/zk/foo:}`. As for files outside of Neorg's workspaces entirely, yeah you have to resort to relative or absolute paths :)

**Reply by pysan3** [reply]
Oh, wow. So amazing!
